src/Model/VideoManager.php

```diff
@@ -2,25 +2,24 @@
 
 namespace Mi\VideoManagerPro\SDK\Model;
 
+use JMS\Serializer\Annotation\Type;
+
 /**
  * @author Steve Reichenbach <steve.reichenbach@movingimage.com>
  *
  * @codeCoverageIgnore
  */
 class VideoManager
 {
+    /**
+     * @Type("string")
```

Comment:
The annotations are already defined here: https://github.com/MovingImage24/videomanager-pro-sdk/blob/master/resources/serializer/VideoManager.xml